packages/orchestration/src/examples/stakeBld.contract.js

```diff
@@ -34,48 +40,86 @@ export const start = async (zcf, privateArgs, baggage) => {
     baggage,
     privateArgs.marshaller,
   );
-  const makeLocalchainAccountKit = prepareLocalchainAccountKit(
+
+  // Mocked until #8879
+  // Would expect this to be instantiated elsewhere, and passed in as a reference
+  const agoricChainInfo = prepareMockChainInfo(zone);
+
+  const timerBrand = await E(privateArgs.timerService).getTimerBrand();
+  const timestampHelper = makeTimestampHelper(
+    privateArgs.timerService,
+    timerBrand,
+  );
```

Comment:
I don't see timerBrand used here. Consider,
```suggestion
  const timestampHelper = await makeTimestampHelper(
    privateArgs.timerService
  );
```

Or if you expect to need timeBrand in the future,

```suggestion
  const { timerBrand, timestampHelper } = await makeTimestampHelper(
    privateArgs.timerService
  );
```

Reply:
> I don't see timerBrand used here.

It's used in `makeTimestampHelper` on line 31 to build the default, right?

https://github.com/Agoric/agoric-sdk/pull/9380/files#diff-31a54bae8709455401dcdf383a90d4eddb79f4916433172da12e13075858e71cR31

Reply:
`makeTimestampHelper`, taking the timerService, can get the brand itself. That's why my examples change it to async